lib/ocamlorg_web/graphql.mli

```diff
@@ -0,0 +1 @@
+val schema : Dream.request Graphql_lwt.Schema.schema
```

Comment:
It might be nice to add a general comment to point out that this schema reflects the data on the opam-repository and allows to query for opam packages. If I'm not missing anything, without a comment it's not directly clear that the graphql endpoint contains information about opam packages.